specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-01-01-preview/Metadata.json

```diff
@@ -628,6 +636,63 @@
       "type": "string",
       "format": "date"
     },
+    "metadataCustomVersion": {
+      "description": "The custom version of the content. A optional free text",
+      "type": "string"
+    },
+    "metadataContentSchemaVersion": {
+      "description": "Schema version of the content. Can be used to distinguish between different flow based on the schema version",
+      "type": "string"
+    },
+    "metadataIcon": {
+      "description": "the icon identifier. this id can later be fetched from the solution template",
+      "type": "string"
+    },
+    "metadataThreatAnalysisTactics": {
+      "description": "the tactics the resource covers",
+      "type": "array",
+      "example": [
+        "reconnaissance",
+        "exfiltration"
+      ],
+      "items": {
+        "type": "string"
+      }
+    },
+    "metadataThreatAnalysisTechniques": {
```

Comment:
>metadataThreatAnalysisTechniques [](http://example.com/codeflow?start=5&length=32)

Same as above, if is a well known list, might be good to model this as an enum

Reply:
Techniques can't be an enum, new techniques are added on a high cadence